lib/fog/profitbricks/compute.rb

```diff
@@ -148,17 +148,22 @@ def request(params)
 
           begin
             response = @connection.request(params)
+            
           rescue Excon::Errors::Unauthorized => error
+            puts 'Unauthorized error'
```

Comment:
You can use `Fog` warnings here instead of plain puts.

See [here](https://github.com/fog/fog-core/blob/f6ef6c3d9458bc235be57100acdced264e85e52a/lib/fog/core/logger.rb#L26)